src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1590,21 +1590,20 @@
    "DROP AGGREGATE IF EXISTS md5_agg(BYTEA)"
    "DROP FUNCTION IF EXISTS dual_md5(BYTEA, BYTEA)"))
 
-(defn autovacuum-vacuum-scale-factor-factsets-catalogs-certnames-reports []
+(defn autovacuum-vacuum-scale-factor-factsets-catalogs-certnames []
   (jdbc/do-commands
    "ALTER TABLE factsets  SET ( autovacuum_vacuum_scale_factor=0.80 )"
    "ALTER TABLE catalogs  SET ( autovacuum_vacuum_scale_factor=0.75 )"
-   "ALTER TABLE certnames SET ( autovacuum_vacuum_scale_factor=0.75 )"
-   "ALTER TABLE reports   SET ( autovacuum_vacuum_scale_factor=0.01 )"))
```

Comment:
Usually we wouldn't remove anything from a past migration... what's the benefit since the partitioning migration will make a new table I don't think this customization will apply to the new reports table?

Reply:
The test for this ensures that at the end of all the migrations, the autovacuum scale factors are applied. With the change to the reports table via partitioning, that value has no effect on the reports table. The statement was removed and the test was updated accordingly.

We have a few other places where we've removed previous migrations or made changes in them because applying them is costly if they're superseded by later migrations.